### ISSUE TITLE
validation torch.no_grad()

### ISSUE THREAD

**Issue by zgojcic** [chrischoy/FCGF#15]
Hi Chris, 

it is not really an issue but when performing the validation you can wrap the code in 
with torch.no_grad():

such that the gradients are not computed. 

It helps reduce the memory consumption and makes the validation faster.

Cheers
Zan

**Comment by chrischoy** [comment]
Hi Zan,

Thanks for the issue. Yes, I'll update the evaluation codes.
